### DISCUSSION TITLE
turbulent inflow PelePhysics

### DISCUSSION THREAD

**Discussion by jagmohas** [AMReX-Combustion/PeleC#621] [Q&A]
I am looking at injecting some turbulence at one of the boundary conditions. Can I use turbinflow from PelePhysics for this? What is the format of the turb file needed for this? Is there any example I can refer to for this?

**Comment by esclapez** [comment]
Hi there,
Indeed not much documentation on that functionality yet. There is an example of code generating a TurbInflow file from an ASCII file containing synthetic turbulence in PeleLMeX:
https://github.com/AMReX-Combustion/PeleLMeX/tree/development/Exec/RegTests/TurbInflow/TurbFileHIT

I'm currently working on extending that piece of code. Where does the data you want as inflow comes from ?

**Reply by jagmohas** [reply]
Thanks esclapez for looking into this. The data will come from another Pelec simulation.
